edge-apps/cap-alerting/src/main.ts

```diff
@@ -0,0 +1,273 @@
+import {
+  setupTheme,
+  signalReady,
+  getMetadata,
+  getTags,
+  getSettings,
+  getSettingWithDefault,
+} from '@screenly/edge-apps'
+
+import { CAPAlert, CAPMode } from './types/cap'
+import { parseCap } from './parser'
+import { CAPFetcher } from './fetcher'
+import { getNearestExit, splitIntoSentences, proxyUrl } from './utils'
+import { highlightKeywords } from './render'
+
+function renderAlerts(
```

Comment:
I feel like the render should be done using a template language, not by using JS.